scout/server/blueprints/variants/forms.py

```diff
@@ -70,7 +70,7 @@ class FiltersForm(FlaskForm):
     gnomad_frequency = BetterDecimalField('gnomadAF', places=2)
     chrom = TextField('Chromosome', [validators.Optional()])
     start = IntegerField('Start position', [validators.Optional(), IntegerField])
-    end = IntegerField('Stop position', [validators.Optional(), IntegerField])
+    end = IntegerField('End position', [validators.Optional(), IntegerField])
```

Comment:
:+1:

Reply:
This is just in place for when we introduce Flask_WTF validate_on_submit. The code is not validated yet.

Reply:
But it is "End" position! Validation can wait. 😄 Besides, there is temporary validation in place - which I ofcourse though you were already calling here as well.. X-)

Reply:
> But it is "End" position! Validation can wait. 😄 Besides, there is temporary validation in place - which I ofcourse though you were already calling here as well.. X-)

I changed into End, didn't you like it more than Stop?

Reply:
Yes! Hence thumbs up! 😆

Reply:
Ah ok, now I see it!